app/views/home/index.html.erb

```diff
@@ -104,7 +104,7 @@
               <td>
                 <span class="overflowing"><%= course.description %></span>
               </td>
-              <td class="text-right"><%= link_to "Join", join_course_path(id: course.id), method: :post, class: "btn btn-primary" %></td>
+              <td class="text-right"><%= link_to "Join", join_course_path(id: course.id), method: :post, class: "btn btn-outline-danger" %></td>
```

Comment:
This should be unchanged join is no danger

Reply:
Correct.